filebeat/filebeat.docker.yml

```diff
@@ -3,6 +3,12 @@ filebeat.config:
     path: ${path.config}/modules.d/*.yml
```

Comment:
ey @DanRoscigno, this is not a sample file, this is the default config we ship with our containers (starting with 7.0), is this change intended?

Reply:
I think that it would be great to have a default file that causes some data to be sent with no configuration, so if we should always get data flowing if the system module is included then I would do that.  But, I did not know that filebeat.docker.yml would be defaulted to in 7, that is great.  I did not intend to change the path if that is what you are asking.  I will take a look in detail tonight, I am in all day meetings and sneaking on to quickly read comments :)